Eastern-Research-Group/expert-query
# Feature/249 display refresh date

## Related Issues:
* [EQ-249](https://jira.epa.gov/browse/EQ-249)
* [EQ-255](https://jira.epa.gov/browse/EQ-255)

## Main Changes:
* Added the date profile data was last refreshed to the home page, under the profile selection input.
* Moved the profile metadata (row count, date refreshed, etc.) into the _Content_ context, since it's being used on multiple pages. Also moved the initial request for the data inside the `lookupFiles` endpoint.
* Moved top-level navigation links into _NavLink_ components from `react-router-dom`. This prevents the page from completely reloading when navigating between pages, but still uses accessible anchor links under the hood.

## Steps To Test:
1. Test the navigation links, and verify they operate as expected and correctly highlight the current page.
2. Confirm that the refresh date is displayed when selecting a profile on the home page.

## TODO:
- [ ] Merge in #86 first, since this incorporates those changes.
